scripts/audit-services.rb

```diff
@@ -1,6 +1,8 @@
 #!/usr/bin/env ruby
 # frozen_string_literal: true
 
+`echo "#{File.absolute_path(__FILE__).sub!(Dir.pwd + "/", "")}" >> ~/.paas-script-usage`
```

Comment:
Feels a bit meh to shell out to echo just to append a line to a file 🤔

Reply:
I was going for the most obvious one-liner that was similar to the one we'd use in the Bash scripts. I'm up for having it replaced with a conventionally Ruby line.

Reply:
```ruby
script_path = File.absolute_path(__FILE__).sub!(Dir.pwd + "/", "")
File.open(File.expand_path('~/.paas-script-usage'), 'a') {|f| f.puts 'script_path' }
```

The `'a'` argument means "append".